tox.ini

```diff
@@ -0,0 +1,40 @@
+[tox]
+envlist= py38,style,bandit
+
+[testenv]
+deps=
+    dev: -rrequirements-dev.txt
+extras= test
+# astropy will complain if the home directory is missing
+passenv= HOME
+commands=
+    pytest
+
+[testenv:style]
+basepython= python3.8
```

Comment:
I think you can leave off defining `basepython` here, and just use the tox env:

```
tox -e py38-style
```

if you care what version of python is used.